Generate.ps1

```diff
@@ -7,7 +7,7 @@ $SWAGGER_PATH = "node_modules/@microsoft.azure/autorest.testserver/swagger"
 $AZURE_DATAPLANE_ARGUMENTS = "--use=./ --output-folder=./azure-dataplane-tests"
 $AZURE_DATAPLANE_PATH = "azure-dataplane-tests/swagger"
 $AZURE_SDK_FOR_JAVA = "https://github.com/Azure/azure-sdk-for-java/blob/main/sdk"
-$PARALLELIZATION = 5
+$PARALLELIZATION = 4
```

Comment:
It's good to document the reason for changing this from 5 to 4.

Reply:
I think I once changed it to 1 but still see it happen.

Anyway, let's try.